README.md

```diff
@@ -27,20 +27,93 @@ The `zone` key is purely static and arbitrary, but by having different `zone` va
 
 ### Run the application
 
-1.  Run `redis` on the default port `6379` and set a default `zone` name:
+1.  Run `redis` 
+
+    - (Kubernetes setup) on the default port `6379`  and set a default `zone` name:
 
     ```sh
     $ redis-server
     $ redis-cli set zone local
     ```
 
-1.  Start `demo-app` on the default port `5000`:
+    - (Universal setup) on port `26379` and set a default `zone` name:
+
+     ```sh
+    $ redis-server --port 26379
+    $ redis-cli set zone local
+    ```
+
+2.  Start `demo-app` on the default port `5000`:
 
     ```sh
     $ npm start --prefix=app/
```

Comment:
On a fresh checkout I needed to do `npm install --prefix=app/`

Reply:
Also the app by default is trying to connect to redis on 6379 which won't work until the dp is started

Reply:
You are right, I will add that step